template.tpl

```diff
@@ -381,7 +381,7 @@ ___TEMPLATE_PARAMETERS___
           }
         ],
         "displayName": "Cookie Name",
-        "defaultValue": "sp",
+        "defaultValue": "_sp_",
```

Comment:
This is a humdinger. Will changing this now, for all those already using `sp` mean that all their cookies get regenerated? Or does the template tell you about this when the upgrade happens in the UI? :-/

Reply:
I was hoping this is pointed by the UI upon available update, but i haven't checked for this specific change. The UI offers a chance to review the changes, which may not be enough as a pointer. So you are right, i don't know if we can be 100% sure. Do you think we should drop this change until a v2 for example?

Reply:
instead of changing this, how about we update the help text to identify this is a different default to the trackers defaults, if you want it to be the same set it to `_sp_`?